test-app/src/widgets/Button.js

```diff
@@ -50,23 +50,9 @@ export default class Button extends createjs.Container {
     this.width = options.width || 300;
     this.height = options.height || 60;
 
-    this.background = new createjs.Shape();
-    this.background.name = 'background';
-    this.background.graphics.beginStroke('black').beginFill('#fff').drawRect(0, 0, this.width, this.height);
-    this.addChild(this.background);
-
-    this.focusRect = new createjs.Shape();
-    this.focusRect.name = 'focusRect';
-    this.focusRect.graphics.beginStroke('black').setStrokeStyle(3).beginFill('#fff').drawRect(0, 0, this.width, this.height);
-    this.addChild(this.focusRect);
-    this.focusRect.visible = false;
-
-    this.text = new createjs.Text(this.value, 'bold 24px Arial', '#000');
-    this.text.textAlign = 'center';
-    this.text.textBaseline = 'middle';
-    this.text.x = this.width / 2;
-    this.text.y = this.height / 2;
-    this.addChild(this.text);
+    this._addBackground();
```

Comment:
Separated these out so that ClearButton can extend Button and override these methods to get the button functionality but keep its unique graphics